buildmaster/old_new_porting_map.yml

```
@@ -0,0 +1,388 @@
+# Mapping between old and new commondata for oldcommondata_porter
+#
+# <old_commondata_name>:
+#   new_name: <new_name>
+#   reference_arxiv: <http://etc>
+#   reference_hepdata: <http://etc>
+#
+
+# HERA
+HERACOMBCCEM:
+  new_name: HERA_CC_318GEV_EM-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t7
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBCCEP:
+  new_name: HERA_CC_318GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t6
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEM:
+  new_name: HERA_NC_318GEV_EM-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t5
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP460:
+  new_name: HERA_NC_225GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t4
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP575:
+  new_name: HERA_NC_251GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t3
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP820:
+  new_name: HERA_NC_300GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t2
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP920:
+  new_name: HERA_NC_318GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t1
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMB_SIGMARED_B:
+  new_name: HERA_NC_318GEV_EAVG_BOTTOM-SIGMARED
+  reference_arxiv: '1804.01019'
+  reference_hepdata: ''
+  journal: Eur. Phys. J. C78 (2018) 473
+HERACOMB_SIGMARED_C:
+  new_name: HERA_NC_318GEV_EAVG_CHARM-SIGMARED
+  reference_arxiv: '1804.01019'
+  reference_hepdata: ''
+  journal: Eur. Phys. J. C78 (2018) 473
+
+## ATLAS
+ATLASLOMASSDY11EXT:
+  new_name: ATLAS_Z0_7TEV_LOMASS_M
+  reference_arxiv: '1404.1212'
+  reference_hepdata: 10.17182/hepdata.64183.v1/t6
+  journal: JHEP 06 (2014) 112
+
+ATLASWRAP11CC:
+  new_name: ATLAS_WPWM_7TEV_CC
+  reference_arxiv: '1612.03016'
+  reference_hepdata: 10.17182/hepdata.76541.v1/t9
+  journal: Eur. Phys. J. C77 (2017) 367
+ATLASZRAP11CC:
+  new_name: ATLAS_Z0_7TEV_46FB_CC
+  reference_arxiv: '1612.03016'
+  journal: Eur. Phys. J. C77 (2017) 367
+ATLASWZRAP11CC:
+  new_name: ATLAS_DY_7TEV_46FB_CC
+  reference_arxiv: '1612.03016'
+  reference_hepdata: 10.17182/hepdata.76541.v1/t9
+ATLASWZRAP11CF:
+  new_name: ATLAS_Z0_7TEV_46FB_CF
+  reference_arxiv: '1612.03016'
+  journal: Eur. Phys. J. C77 (2017) 367
```

I assume these are all differential in pseudorapidity.

```suggestion
ATLASZRAP11CC:
  new_name: ATLAS_Z0_7TEV_46FB_CC-ETA
  reference_arxiv: '1612.03016'
  journal: Eur. Phys. J. C77 (2017) 367
ATLASWZRAP11CC:
  new_name: ATLAS_DY_7TEV_46FB_CC-ETA
  reference_arxiv: '1612.03016'
  reference_hepdata: 10.17182/hepdata.76541.v1/t9
ATLASWZRAP11CF:
  new_name: ATLAS_Z0_7TEV_46FB_CF-ETA
  reference_arxiv: '1612.03016'
  journal: Eur. Phys. J. C77 (2017) 367
```

---

I think the distributions we have are in  rapidity (of the gauge boson / dilepton pair) rather than pseudorapidity. But the point still stands, I'll add `-Y` to these.

(I'll check first whether they are η or y)
thanks!

---

For Z it's the rapidity of the leptonpair and for W the pseudorapidity of the visible lepton.

Yet another datapoint in favour of separating W and Z :P 

@enocera I'm going to put `-Y` for Z and `-ETA` for W and nothing for `DY`. Are you happy with this choice or you'd rather do something different?

---

Happy am I.

---

Ok, good. Thanks